### ISSUE TITLE
Watch repo suggestion

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/topcoder-x-ui#84]
From a Slack discussion:

> A suggestion on the new topcoder-x tool which is used to subscribe to repos for challenges. Unfortunately it’s also subscribing the users to the forks as well. It’s ok to subscribe to the main repo. But I would think it’s not necessary to be subscribed to all the forked repos as well.

**Comment by jmgasper** [comment]
@rootelement - We'll have to investigate this as part of a challenge.  This must be the default in Gitlab / Github, but I'm not entirely sure we can change that behaviour.

I'll ask if a member can take a look at it in the short term just to see if it's feasible.

**Comment by jmgasper** [comment]
From the latest challenge:

I did some investigation to identify the problem and the possible solution.

Here are the points
1. The behavior happens in Github.
   I have compared my teams on Github and my groups on Gitlab that i've joined with Topcoder-X tools.
   The issue is confirmed that my Github account automatically subscribe all of forked repositories from a repo that i've joined the team by Topcoder-X. The repository and all of the forked repositories are private and I've access on all of them.
   On the other hand, the issue doesn't appear in Gitlab. My Gitlab account doesn't subscribe to forked repositories and doesn't have access to them.
  https://drive.google.com/open?id=1zxZdTdieIskjQ8SUB4CdhdexLnRi1ZDh   
  https://drive.google.com/open?id=1ewoF6xYoEPoKl4TiVS3As9RoIAmZIHQ0

2. Topcoder-X use teams/groups for inviting users instead of single repository.
   Since Topcoder-X use teams for inviting users, the invited user will automatically subscribe and has access to any repositories includes the forked repository. This is the default behavior of Github.

3. We can change in settings.
   According to a forum conversation, the can change the behavior by configuring in setting page.
   https://webapps.stackexchange.com/questions/62472/stop-being-auto-subscribed-to-github-repositories-from-an-organisation
   The link of Github setting is https://github.com/settings/notifications.
